onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -305,55 +310,6 @@ def concat_kv(self, past_k: str, past_v: str) -> str:
 
         return kv_output_name
 
-    def reshape_kv(self, past_k: str, past_v: str) -> (str, str):
```

Comment:
Should we also remove `transpose_kv`? Both `reshape_kv` and `transpose_kv` were originally added for Whisper.